tests/baselines/reference/user/webpack.log

```diff
@@ -0,0 +1,10 @@
+Exit Code: 1
+Standard output:
+lib/Compilation.js(575,5): error TS2322: Type 'number | "toString" | "valueOf" | "charAt" | "charCodeAt" | "concat" | "indexOf" | "lastIndexOf" | "localeCompare" | "match" | "replace" | "search" | "slice" | "split" | "substring" | ... 29 more ... | "trimRight"' is not assignable to type 'string'.
```

Comment:
OK, so this is broken because of webpack. They wrote `/** @typedef {keyof LogType} LogTypeEnum */`. `LogType` is `string`. That is `keyof string`. That is wrong. They need to write, instead, `/** @typedef {keyof typeof LogType} LogTypeEnum */`. Not really sure why this doesn't error in however they're testing their types, but it should.

cc @sokra just so yall know